File: push/src/main/java/com/xiaomi/mipush/sdk/help/HelpActivity.java
package com.xiaomi.mipush.sdk.help;

import android.app.Activity;
import android.os.Bundle;
import com.xiaomi.mipush.sdk.AwakeHelper;

/* JADX INFO: loaded from: miuipushsdkshared_3_7_9.jar:com/xiaomi/mipush/sdk/help/HelpActivity.class */
public class HelpActivity extends Activity {
    @Override // android.app.Activity
    protected void onCreate(Bundle bundle) {
        super.onCreate(bundle);
        AwakeHelper.doAWork(this, getIntent(), null);
        finish();
    }
}

File: xmsf/shell/src/main/java/com/xiaomi/mipush/sdk/help/HelpActivity.kt
package com.xiaomi.mipush.sdk.help

import android.app.Activity
import android.os.Bundle
import com.xiaomi.mipush.sdk.AwakeHelper

class HelpActivity : Activity() {
    override fun onCreate(bundle: Bundle?) {
        super.onCreate(bundle)
        AwakeHelper.doAWork(this, intent, null)
        finish()
    }
}
